README.rst

```diff
@@ -49,6 +49,19 @@ Build the documentation::
 
    ninja -C build doc/html
 
+Installing
+==========
+
+Install partup on the current system. If enabled, the documentation will be
+installed as well::
+
+   meson install -C build
+
+Yocto Integration
+=================
+
+A Yocto recipe is available in `meta-ampliphy <https://github.com/phytec/meta-ampliphy/tree/master/recipes-devtools/partup>`_.
```

Comment:
Please use our repository at [git.phytec.de](https://git.phytec.de). The one on GitHub is a mirror.